code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -432,3 +432,35 @@
 /mob/living/carbon/human/proc/delete_equipment()
 	for(var/slot in get_all_slots())//order matters, dependant slots go first
 		qdel(slot)
+
+/mob/living/carbon/human/proc/quick_equip_bag() // puts things in bag
+	var/obj/item/thing = get_active_hand()
+	var/obj/item/storage/equipped_backpack = get_item_by_slot(slot_back)
+	if(ismecha(loc) || HAS_TRAIT(src, TRAIT_HANDS_BLOCKED))
+		return
+	if(!istype(equipped_backpack)) // We also let you equip a backpack like this
+		equip_to_slot_if_possible(thing, slot_back)
+		return
+	if(thing && equipped_backpack.can_be_inserted(thing)) // put thing in backpack
+		equipped_backpack.handle_item_insertion(thing)
+		playsound(loc, "rustle", 50, 1, -5)
+
+/mob/living/carbon/human/proc/quick_equip_belt() // puts things in belt
+	var/obj/item/thing = get_active_hand()
+	var/obj/item/storage/equipped_belt = get_item_by_slot(slot_belt)
+	if(ismecha(loc) || HAS_TRAIT(src, TRAIT_HANDS_BLOCKED))
+		return
+	if(!istype(equipped_belt)) // We also let you equip a belt like this
+		equip_to_slot_if_possible(thing, slot_belt)
+		return
+	if(thing && equipped_belt.can_be_inserted(thing)) // put thing in belt
+		equipped_belt.handle_item_insertion(thing)
+		playsound(loc, "rustle", 50, 1, -5)
```

Comment:
This code is somewhat duplicated. Think you could consolidate it to a general quick_equip proc and pass in the slot you want to equip things to?

Reply:
How would you be able to rebind it then? They all need to be a separate proc for each bind

Reply:
Actually, you could just use the same proc but pass a different arg for the slot depending on the keybind

Reply:
should be better now, based on what Vi3trice recommended I should do